src/response/mod.rs

```diff
@@ -130,11 +130,18 @@ impl Response {
     /// # Mime
     ///
     /// The encoding is set to `application/json`.
+    #[cfg(not(feature = "simd-json"))]
     pub fn body_json(mut self, json: &impl Serialize) -> serde_json::Result<Self> {
         *self.res.body_mut() = serde_json::to_vec(json)?.into();
         Ok(self.set_mime(mime::APPLICATION_JSON))
     }
```

Comment:
Why not just
```rust
    pub fn body_json(mut self, json: &impl Serialize) -> serde_json::Result<Self> {
        #[cfg(not(feature = "simd-json"))]
        *self.res.body_mut() = serde_json::to_vec(json)?.into();
        #[cfg(feature = "simd-json")]
        *self.res.body_mut() = simd_json::to_vec(json)?.into();
        Ok(self.set_mime(mime::APPLICATION_JSON))
    }
```

Reply:
That's not legal rust at the moment, attributes on expressions are not supported :)